system/lib/pthread/pthread_create.c

```diff
@@ -77,6 +76,12 @@ int __pthread_create(pthread_t *restrict res, const pthread_attr_t *restrict att
     return EINVAL;
   }
 
+  // Create threads with monotonicly increasing TID starting with the main
+  // thread which has TID == PID.
```

Comment:
how are PIDs determined?

Reply:
Its defined in JS: 

https://github.com/emscripten-core/emscripten/blob/b2cfa354366536963b8e935469b20bb5d0843c0d/src/runtime.js#L64-L69

https://github.com/emscripten-core/emscripten/blob/b2cfa354366536963b8e935469b20bb5d0843c0d/src/library_syscall.js#L338-L340